scripts/monitor/rano_monitor/handlers/reviewed_handler.py

```diff
@@ -1,115 +0,0 @@
-import os
-import re
-import tarfile
-
-from rano_monitor.constants import (
-    BRAINMASK_PATTERN,
-    REVIEW_FILENAME,
-    REVIEWED_PATTERN,
-)
-from rano_monitor.utils import delete
-from watchdog.events import FileSystemEventHandler
-
-
-def get_tar_identified_masks(file):
-    identified_reviewed = []
-    identified_brainmasks = []
-    try:
-        with tarfile.open(file, "r") as tar:
-            for member in tar.getmembers():
-                review_match = re.match(REVIEWED_PATTERN, member.name)
-                if review_match:
-                    identified_reviewed.append(review_match)
-
-                brainmask_match = re.match(BRAINMASK_PATTERN, member.name)
-                if brainmask_match:
-                    identified_brainmasks.append(brainmask_match)
-    except Exception:
-        return [], []
-
-    return identified_reviewed, identified_brainmasks
-
-
-def get_identified_extract_paths(
-        identified_reviewed,
-        identified_brainmasks,
-        dset_data_path
-):
-    extracts = []
-    for reviewed in identified_reviewed:
-        id, tp, filename = reviewed.groups()
-        src_path = reviewed.group(0)
-        dest_path = os.path.join(
-            dset_data_path,
-            "tumor_extracted",
-            "DataForQC",
-            id,
-            tp,
-            "TumorMasksForQC",
-            "finalized",
-        )
-        if not os.path.exists(dest_path):
-            # Don't try to add reviewed file if the dest path
-            # doesn't exist
-            continue
```

Comment:
sorry I thought I already made a comment. just want to point out that this block of code is removed when you moved this logic to `utils`

Reply:
Not sure why I decided to remove that, but I'll add it back to all the identified cases. Thanks for checking that!